source/_integrations/iammeter.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: "iammeter"
+description: "Instructions on how to integrate iammeter sensor within Home Assistant."
```

Comment:
Seems like `iammeter` is written in all caps, when looking at their website and documentation.

This needs to be corrected throughout this document (with the exception of the platform in the code example of course).

```suggestion
description: "Instructions on how to integrate IAMMETER sensor within Home Assistant."
```